openmp/runtime/src/z_Linux_asm.S

```diff
@@ -19,6 +19,16 @@
 
 #if KMP_ARCH_X86 || KMP_ARCH_X86_64
 
+# if defined(__ELF__) && defined(__CET__) && defined(__has_include)
+# if __has_include(<cet.h>)
+# include <cet.h>
+# endif
+# endif
+
+# if !defined(_CET_ENDBR)
+# define _CET_ENDBR
+# endif
```

Comment:
This is adopted from https://github.com/llvm/llvm-project/blob/9e863cd44945345f22a28cdd3ea12aaa7963345e/llvm/lib/Support/BLAKE3/blake3_avx2_x86-64_unix.S#L9-L17 We have a couple minor variants of this in different asm files.